### ISSUE TITLE
Updated goals for Kale

### ISSUE THREAD

**Issue by brendanjmeade** [brendanjmeade/kale#12]
@banesullivan Some important notes about visualization with `kale`:

- **TLDR**: I think *time-dependent* interactive visualization isn't the way to go because it's way too slow and crashes (see `kale_crash.png`).  Instead, I think we should focus on two things: 1) Animations, which are currently working great! and 2) a new way to cleanly plot results at a given time step as figures that do not have time interactivity.  
  
- **Why I think this**: The size of model outputs is much larger than anticipated.  Some model runs are now producing multi-TB data sets.  Even a ~200 GB data set runs for 5 minutes before crashing (see `kale_crash.png`).  However, rendering an animation of this same data works very, very well!  See `kale_nankai.ipynb` for an example.  I've uploaded the relavant .vtk and .hdf input files to our shared Google Drive folder: https://drive.google.com/drive/folders/1-vDHHNPK1X013vFrn_NSXnBDCF5hB0mJ?usp=share_link
  
- **What we should do**:  For data exploration and publication-quality figures, we want the ability to render a single figure very nicely (see `new_kale_static.png`).  I think we can do this with [a combination of Pyvista and Matplotlib](https://docs.pyvista.org/examples/02-plot/chart_overlays.html) .
  
- **How to do this**: Create a script that allows us to specify the following:
    - [A lot of this is done](https://github.com/brendanjmeade/kale/blob/main/kale_single_step_erosion_figures_nankai.ipynb)
	- ~~vtk file name for mesh~~
	- ~~.hdf file name for scalars~~
	- .pickle file for time series of events (lower panel in `new_kale_static.png`).  This contains indices and magnitudes of individual events.  Still need to combine matplotlib and pyvista in a single figure.
	- ~~time step to plot~~
	- ~~name of field to plot~~
	- ~~color map levels~~ 
	- ~~colormap~~
	- ~~camera position~~

<img width="1449" alt="kale_crash" src="https://user-images.githubusercontent.com/4225359/223296676-962a7115-27a3-43ad-abaa-79d967cf2817.png">

![new_kale_static](https://user-images.githubusercontent.com/4225359/223296684-69cb0359-23d5-44c4-b38e-ef940fab0dc4.png)

**Comment by brendanjmeade** [comment]
@banesullivan I've updated kale_single_step_erosion_figures_nankai.ipynb with the matplotlib stem plot that should be combined with the banded contour plot.  I just don't know how to put them together in one plot!

**Comment by banesullivan** [comment]
@brendanjmeade, I'm missing `skies_single_step_erosion_figures_nankai.npz` to run this notebook

**Comment by brendanjmeade** [comment]
@banesullivan Just uploaded it to the shared Kitware Google Drive folder!

**Comment by brendanjmeade** [comment]
I think that everything here has been completed or rolled into other issues.
